### ISSUE TITLE
[Feature Request] upvote/downvote Recommendations like in the website

### ISSUE THREAD

**Issue by curche** [zend10/AL-chan#31]
So, as you may know, the Anilist website provides recommendations for anime/manga that you're viewing and also allows you to option to upvote/downvote (thumbsup and thumbsdown) the recs to let it know whether it is a good suggestion or not.

Would it be possible to integrate it into the app? Not sure if it is already present although I did check for it but in vain.

I've been using the app for a few weeks.
Thought it would be a nice feature to have in the app

**Comment by zend10** [comment]
Yes, it's possible. I was just not prioritizing it because it didn't seem like a very useful feature. I will most likely implement it one day.

**Comment by curche** [comment]
Okay, Glad to hear that.
Will wait for it.
